src/index.html

```diff
@@ -18,5 +31,81 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="images"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
+              href="Apple"
+              class="nav__link is-active"
+            >
+              APPLE
+            </a>
+          </li>
+          <li>
+            <a
+              href="Samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li>
+            <a
+              href="Smartphones"
```

Comment:
The `href` attribute should contain a valid URL or path. Currently, it is set to 'Smartphones', which is not a valid URL. Consider using a relative path like './smartphones.html' or an absolute URL.